fintech-examples/fintech-ui/src/app/bank/payment/payment-account-payments/payment-account-payments.component.ts

```diff
@@ -12,30 +17,57 @@ import { Consts } from '../../../models/consts';
 export class PaymentAccountPaymentsComponent implements OnInit {
   public static ROUTE = 'payments';
   list: PaymentInitiationWithStatusResponse[];
+  ibanForm: FormGroup;
+  isRandomAccountId: boolean;
+  bankId: string;
+  accountId: string;
 
   constructor(
     private router: Router,
     private route: ActivatedRoute,
+    private formBuilder: FormBuilder,
+    private location: Location,
+    private storageService: StorageService,
     private fintechRetrieveAllSinglePaymentsService: FintechRetrieveAllSinglePaymentsService
-  ) {}
+  ) {
+  }
 
   ngOnInit() {
-    const bankId = this.route.snapshot.params[Consts.BANK_ID_NAME];
-    const accountId = this.route.snapshot.params[Consts.ACCOUNT_ID_NAME];
-    this.fintechRetrieveAllSinglePaymentsService
-      .retrieveAllSinglePayments(bankId, accountId, '', '', 'response')
-      .pipe(map((response) => response))
-      .subscribe((response) => {
-        this.list = response.body;
-      });
+    this.bankId = this.route.snapshot.params[Consts.BANK_ID_NAME];
+    this.accountId = this.route.snapshot.params[Consts.ACCOUNT_ID_NAME];
+
+    this.isRandomAccountId = this.accountId === Consts.RANDOM_ACCOUNT_ID;
```

Comment:
Making payment without account ID should not be tied to some `random account id`. It should be a simple feature toggle inside FinTech server